### ISSUE TITLE
[BUG] - Lidarr - Unraid Mover Errors

### ISSUE THREAD

**Issue by Kamology** [RandomNinjaAtk/arr-scripts#237]
**Application**
Lidarr

**Host platform**
Unraid

**Script**
Lidarr Extended

**Script Version**
2.35

**Describe the bug**
Unraid "Mover" isn't moving audio files (.flac in my case)

**To Reproduce**
1. Let script download music
2. let Unraid try to move files

**Expected behavior**
All files get moved to their designated destination as configured in Unraid

**Logs/Screenshots**

```
Apr 11 23:18:43 UnraidServer move: move_object: /mnt/server/media/processed/Music/Don Toliver/Love Sick/01 - No Pole.flac File exists
Apr 11 23:18:43 UnraidServer move: file: /mnt/server/media/processed/Music/Don Toliver/Love Sick/02 - Embarrassed.flac
Apr 11 23:18:43 UnraidServer move: move_object: /mnt/server/media/processed/Music/Don Toliver/Love Sick/02 - Embarrassed.flac File exists
Apr 11 23:18:43 UnraidServer move: file: /mnt/server/media/processed/Music/Don Toliver/Love Sick/03 - Geronimo.flac
Apr 11 23:18:43 UnraidServer move: move_object: /mnt/server/media/processed/Music/Don Toliver/Love Sick/03 - Geronimo.flac File exists
Apr 11 23:18:43 UnraidServer move: file: /mnt/server/media/processed/Music/Don Toliver/Love Sick/04 - Luckily I'm Having.flac
Apr 11 23:18:43 UnraidServer move: move_object: /mnt/server/media/processed/Music/Don Toliver/Love Sick/04 - Luckily I'm Having.flac File exists
```
![image](https://github.com/RandomNinjaAtk/arr-scripts/assets/65583898/01de44bc-6674-46d7-9c9b-7a54d5ac31d1)
^ "Location" in all of these should be disk6 or any other "disk"
![image](https://github.com/RandomNinjaAtk/arr-scripts/assets/65583898/9b3785b2-72b4-466f-b90a-195fb5718e4f)

**Additional context**
This very well may be an Unraid configuration error or a user configuration error with the script, but if so I'm not really sure how to fix. It appears the .flac files aren't being moved because the "File exists"... but I don't really know what that means in this context because they don't exist where I want them to be and, theoretically, where Unraid knows they're supposed to be. 

My download/import flow, including Unraids Mover, is:
1 (Lidarr/script handles this step). Download to Lidarr's appdata folder (which is located on my "server" NVMe drive, where all my appdata folders are)
3 (Lidarr/script handles this step). Move/"import" the files from appdata to their proper music library destination ("media" share)
2 (Unraid handles this step). Move the downloaded files off of the "server" NVMe drive (their original download location in appdata folder) to the main array drives (which is where all of the non .flac files are successfully being transferred to (disk6))

But instead, the .flacs get stuck behind on the "server" drive instead of the main array. If you're not familiar with Unraid it can be a little confusing so feel free to ask clarification if this isn't clear enough.

**Comment by RandomNinjaAtk** [comment]
Nothing to be done on this end. But it looks like it’s telling you the file already exists in the destination location. So maybe try deleting those files to clear the error. 

Also consider using the unraid forum to ask for help/info on the error.

Please report back any findings you have that could be useful to others, but since there isn’t anything to do on this end, I’m closing this issue.

**Comment by Kamology** [comment]
Was indeed an Unraid error (Unraid can't handle files that get moved across shares that have different cache/array settings). 

That being said, the default config for the extended script basically induces this broken behavior in Unraid unless you manually adjust the paths (default storage location is in `config` which is almost always mapped to `appdata` in Unraid, and you shouldn't be using `appdata` for storage). So, not a script problem, technically, but could be worth a disclaimer for Unraid users maybe. 

Unraid discussion for reference: https://forums.unraid.net/topic/162614-mover-not-moving-files-because-file-exists/

Also, tagging this on here as an extra note, the script still appears to append `/audio/complete` and `/audio/incomplete` to custom paths I set in `extended.conf`, specifically to the `downloadPath` I believe. I set it to `/path/incomplete` but it downloads in `/path/incomplete/audio/incomplete`, however it does successfully move the files to the path I set for `importPath`, in this case, `/path/complete`.

It doesn't *really* matter, everything works for me now, but figured I'd mention it. Just makes the path more convoluted than intended.
